index.js

```diff
@@ -5,11 +5,11 @@ const PLUGIN_NAME = 'Alarmdotcom'
 const MANUFACTURER = 'Alarm.com'
 const AUTH_TIMEOUT_MINS = 10 // default for session authentication refresh
 const POLL_TIMEOUT_SECS = 60 // default for device state polling
-const LOG_LEVEL = 3  // default for log entries: 0 = NONE, 1 = ERROR, 2 = WARN, 3 = NOTICE, 4 = VERBOSE
+const LOG_LEVEL = 4  // default for log entries: 0 = NONE, 1 = ERROR, 2 = WARN, 3 = NOTICE, 4 = VERBOSE
```

Comment:
Should this be kept at 3?

Reply:
The default is 3, because I figure when people install the plugin for the first time, they want to see the successful logging in and initialization of their sensors. Obviously, they can add the configuration to turn these off when they feel confident that everything is working correctly.